Torpedo.VideoConverter/Video/XabeConverter.cs

```diff
@@ -24,10 +24,11 @@ static XabeConverter()
             {
                 json = r.ReadToEnd();
             }
-            JToken jObj = JToken.Parse(json);
+            UserSetting userSetting = JsonSerializer.Deserialize<UserSetting>(json);
```

Comment:
Issue the same as on Voice recognition. 
If the same issue occurs in two different places, you need to do smth so that it can be solved in one place 😉